core/types/validator.go

```diff
@@ -87,3 +87,15 @@ func (set *ValidatorSet) Contains(addr common.Address) bool {
 	_, ok := set.membership[addr]
 	return ok
 }
+
+func NewDeposit(amount *big.Int) *Deposit {
+	return &Deposit{
+		amount: amount,
+	}
+}
+
+// Deposit represents the validator deposits at stake
+type Deposit struct {
+	amount *big.Int
+	//releasedAt
```

Comment:
Any reason why `amount` is private? this type is returned by the `GetDeposits`, if we don't expose it somehow it's not very useful to get the deposits. Also, any reason why the `ReleasedAt` is not included?